fs/dafsserver/dafsserver.cpp

```diff
@@ -3424,7 +3424,8 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
     Owned<ISocket>      acceptsock;
     Owned<ISocket>      securesock;
     Owned<ISocket>      rowServiceSock;
-
+    Linked<IPropertyTree> componentConfig;
+    bool directIO = true;
```

Comment:
minor: should this default to false for slightly better security (default to restricted) - although I suspect it is that way for bare-metal defaults?

Reply:
it's default to on for the bare metal case, could reverse it and switch it on if #ifndef _CONTAINERIZED